src/main/java/uk/gov/pay/adminusers/service/InviteFinder.java

```diff
@@ -32,6 +33,21 @@ public InviteFinder(InviteDao inviteDao, UserDao userDao) {
                         return Optional.of(invite);
                     }).orElse(Optional.of(invite));
                 })
-                .orElseGet(() -> Optional.empty());
+                .orElseGet(Optional::empty);
+    }
+
+
+    public List<Invite> findAll(String serviceId) {
```

Comment:
Is `findAll` the best name given it filters out expired and disabled invites?

Reply:
💯 for rethorical question